tss-esapi/src/context/tpm_commands/duplication_commands.rs

```diff
@@ -1,9 +1,57 @@
 // Copyright 2021 Contributors to the Parsec project.
 // SPDX-License-Identifier: Apache-2.0
-use crate::Context;
+use crate::{
+    handles::ObjectHandle,
+    structures::{Data, EncryptedSecret, Private, SymmetricDefinitionObject},
+    tss2_esys::*,
+    Context, Error, Result,
+};
+use log::error;
+use mbox::MBox;
+use std::convert::TryFrom;
+use std::ptr::null_mut;
 
 impl Context {
-    // Missing function: Duplicate
+    /// Change authorization for a TPM-resident object.
+    pub fn duplicate(
```

Comment:
Note: this function isn't used anywhere, I added it because I thought I might use it but ended up relying on `ObjectChangeAuth` instead.

Reply:
If you don't mind I'd suggest removing it then.

I will add it as part of the https://github.com/parallaxsecond/rust-tss-esapi/pull/248 PR including tests. Sorry that it takes some time but I've got a work schedule but duplicate is now my next work item and I also found integration test in C that may help me get the test in shape: https://github.com/utkarsh2102/tpm2-tss/blob/5eca0a4ae129f410303de1c20046ebe7527b9ad3/test/integration/esys-duplicate.int.c#L382

:wave:

Reply:
I'll remove it, then.

Reply:
Thank you :bow:

Now I remembered that Jesper also brought interesting point that new context functions could use some more improved documentation (including working examples): https://github.com/parallaxsecond/rust-tss-esapi/pull/255#discussion_r704571544 so this would also be problematic here. I'll keep that in mind in my PR.

Reply:
> new context functions could use some more improved documentation

Yeah, I was thinking of that... 😬